handycon.py

```diff
@@ -662,6 +668,45 @@ async def capture_keyboard_events():
                             elif active == [] and seed_event.code == 125 and button_on == 0 and  event_queue == [] and shutdown == True:
                                 shutdown = False
 
+                        case "AOK":
```

Comment:
Are the key combinations for these buttons different than the OXP?

Reply:
I tested it and it was normal

Reply:
Looking at the new code and the OXP, it doesn't appear there are any differences. It would be much cleaner to just add the AOKZOE A1 AR07 to the OXP list rather than add an entire identical block.

Reply:
I have a branch that has just this,  but with the one tester I've had it didn't work as expected. I haven't done anything with it yet because I'm confused by the difference in experiences.

Reply:
Are there different firmware's in the wild?

Reply:
It's very well possible. I say for now we can just merge it and if it works for some thats better than none I suppose.

Reply:
Alright. I'll close this one for now and we can revisit if needed.

I think we should actually keep the AOK vs OXP detection, because I'd like to change the sleep timer based on that. We can just add AOK to the case block for OXP.

Reply:
On AOK, the home button is interchangeable with the long press and short press

Reply:
I'm glad you're back. I'm not a professional programmer. I'm actually a communications engineer